src/python/bcc/__init__.py

```diff
@@ -1123,6 +1123,18 @@ def support_raw_tracepoint():
             return True
         return False
 
+    @staticmethod
+    def support_raw_tracepoint_module():
+        # kernel symbol "bpf_trace_modules" indicates raw tp module support, ref: kernel commit a38d1107
```

Comment:
Let us name it as `support_raw_tracepoint_in_module` to be more clear what we are testing here.
So "indicates raw tp module support" => "indicates raw tp support in modules".